agent/js/src/wd_server.js

```diff
@@ -155,7 +155,8 @@ WebDriverServer.prototype.initIpc = function() {
  */
 WebDriverServer.prototype.init = function(args) {
   'use strict';
-  args.startURL = args.task['Capture Video'] ? ORANGE_PAGE_URL_ : BLANK_PAGE_URL_;
+  args.startURL = args.task['Capture Video'] && !args.task.lighthouse ? 
+      ORANGE_PAGE_URL_ : BLANK_PAGE_URL_;

```

Comment:
In the case of video capture, the orange page needs to be up at the time of the start of the capture.  Unless lighthouse is going to run the screenrecord, the page needs to be loaded before lighthouse starts.